profiles/default/kafka/Dockerfile.ftl

```diff
@@ -2,6 +2,11 @@ FROM docker.io/bitnami/kafka:3.4.0-debian-11-r38
 
 RUN echo '#!/bin/bash' > /opt/bitnami/scripts/kafka/create-topics.sh \
 <#list kafka["topics"] as topic>
-    && echo '/opt/bitnami/kafka/bin/kafka-topics.sh --create --bootstrap-server kafka:9092 --topic ${topic.name} --partitions ${topic.numPartitions} --replication-factor ${topic.replicationFactor}' >> /opt/bitnami/scripts/kafka/create-topics.sh \
+    && echo '/opt/bitnami/kafka/bin/kafka-topics.sh --create --if-not-exists --bootstrap-server kafka:9092 --topic ${topic.name} --partitions ${topic.numPartitions} --replication-factor ${topic.replicationFactor}' >> /opt/bitnami/scripts/kafka/create-topics.sh \
 </#list>
+<#if config["values"]?? && config["values"]["addl-topics"]??>
```

Comment:
Can we please rename this option to `create-topics`? It's a little more obvious imho.